src/index.js

```diff
@@ -72,15 +72,34 @@ const getLooseMode = looseOpts => {
   return looseOpts;
 }
 
-// TODO: Allow specifying modules as: Boolean, String or Array of module types
-const getModules = modulesOpts => {
+/**
+ * Given modulesOpts return an Array of all required module transformations
+ * @param  {String|Array.<String>|Boolean}  modulesOpts  An argument enumerating the modules to transform
+ * @return {Array.<String>}  An array of module transformation plugin names
+ */
+export const getModuleTransformations = modulesOpts => {
   if (!modulesOpts) { return []; }
-  return modulesOpts;
+
+  if (modulesOpts === true) { return Object.values(MODULE_TRANSFORMATIONS); }
```

Comment:
Is there some incompatibility between the transformations? I assumed it would be possible to turn all of them on and transform any module type it encountered. I usually do module transformations in my bundler (require.js/webpack/browserify) so it's very possible that I misunderstood how Babel does it.